konfera/views.py

```diff
@@ -13,3 +13,17 @@ def sponsor_list_view(request, event_slug):
     return render(request=request,
                   template_name='konfera/event_sponsors.html',
                   context=context, )
+
+
+def event_speakers_list_view(request, event_slug):
+    event = get_object_or_404(Event, slug=event_slug)
+    talks = Talk.objects.select_related('event')\
```

Comment:
Try to get all talks via Event like few lines above `sponsors = event.sponsors.all()`. Its simple and better readable :)